docs/source/index.md

```diff
@@ -1,86 +1,62 @@
 
-When registration has completed, you should be able to navigate
-[here](https://console.latch.bio/workflows) and see your new workflow in your
-account.
+## Problem Latch SDK does not yet solve 
+* <b>Workflows chaining</b>: We'll aim to support easy installation and reuse of other Latch SDK workflows in your own workflow.  
+* <b>Workflows monitoring</b>: For batched runs of workflows, we'll aim to provide better dashboard, logs, traces, metrics, and alerting for observability. 
 
-If you are having issues with registration or have general questions, please
-file an issue on [github](https://github.com/latchbio/latch).
+## What Latch SDK is not
+* <b>A pure workflow orchestration engine</b>: There are many popular workflow orchestration engines, such as Nextflow or Snakemake, that can be run locally from a bioinformatician's machine. Although workflow orchestration is a feature of Latch SDK, Latch is fundamentally a cloud-native solution. Latch SDK solves the problems of automating pipelines by providing a robust and scalable cloud infrastructure and asssociated no-code interfaces out of the box. You can also easily bring existing workflow script of any language to Latch (See examples [here](./examples/)).
+* <b>A self-hosted solution</b>: Currently, you cannot write your workflow using Latch SDK and host it in your own AWS instance or an HPC. The infrastructure serving bioinformatics pipelines is fully managed by Latch. This allows us to rapidly iterate to bring on high quality features, give cost and performance guarantees, and ensure that security is offered out-of-the-box. 
 
----
+<hr/>
 
-### Installation
+## Next Steps
 
-The SDK is distributed on pip. Install in a fresh virtual environment for best
-behavior.
-
-[Virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/) is recommended.
-
-```bash
-python3 -m pip install latch
-```
-
-_Note that a local installation of docker is required to register workflows_.
+To get started with Latch SDK, view the following resources:
+* <b>[Quickstart](./getting_started/quick_start.md)</b> is the fastest way to get started with the Latch SDK. 
+* <b>[Concepts](./basics/what_is_a_workflow.md)</b> describes all important Latch SDK concepts.
+* <b>[Examples](./examples/workflows_examples.md)</b> show full examples of using Latch SDK for various bioinformatics pipelines.
+* <b>[Troubleshooting](./troubleshooting/troubleshooting)</b> provides a guide to debug common errors. 
+* <b>[Reference](./api/modules.rst)</b> contains detailed API and design documents.
+* <b>[Subcommands](./subcommands.md)</b> contains details about the Latch command line toolchain to register workflows and upload data to Latch.
+* Join the <a href="https://forms.gle/sCjr8tdjzx5HjVW27" target="_blank">SDK open-source community</a> on Slack!
 
 ---
 
-### Examples
```

Comment:
i still want these on the front page, maybe link to the section page you made

Reply:
I linked that in Next steps. Do you want to make it big and center?